modules/calib3d/test/test_homography.cpp

```diff
@@ -144,7 +144,7 @@ void CV_HomographyTest::print_information_1(int j, int N, int _method, const Mat
     cout << "Type of srcPoints: "; if ((j>-1) && (j<2)) cout << "Mat of CV_32FC2"; else  cout << "vector <Point2f>";
```

Comment:
@vpisarev Pardon? When the tests succeed they print almost nothing at all.

{code}
[----------] 1 test from Calib3d_StereoSGBM (1112 ms total)

[----------] 1 test from Calib3d_Homography
[ RUN      ] Calib3d_Homography.accuracy
[       OK ] Calib3d_Homography.accuracy (4533 ms)
[----------] 1 test from Calib3d_Homography (4533 ms total)

[----------] 6 tests from OCL_StereoMatcher/StereoBMFixture
{code}